src/Mod/Sketcher/App/SketchObject.cpp

```diff
@@ -426,7 +426,7 @@ const std::map<std::string,std::string> SketchObject::getInternalElementMap() co
     if (!internalElementMap.empty() || !MakeInternals.getValue())
         return internalElementMap;
 
-    auto internalShape = InternalShape.getShape();
+    auto& internalShape = InternalShape.getShape();

```

Comment:
If it was a copy and now is reference - it should be immutable reference probably?